### ISSUE TITLE
Not being able to open files in Collabora Online that don't have share permissions

### ISSUE THREAD

**Issue by jothoma1** [nextcloud/groupfolders#445]
The Fix of 4.0.1 did not work we fully upgrade our cluster and we are on the 4.0.1 release but still cant edit with collabora...

**Comment by langfingaz** [comment]
Could you please clarify your problem? I just did a short test:
Alice created text document in group folder that Bob can only read and not reshare or edit. If Bob accesses the document it is opened with Collabora Online.

Greetings,
Daniel

![no-reshare](https://cloud.privacy1st.de/s/jkLMY4iqQKFTCMt/preview)

**Comment by jothoma1** [comment]
Yes of course. I searched a bit and figure why it dont work. My problem is that i have users on group folders that have write permissions (but no share) and collabora wont open in edit mode (just read only) so they cant modify the doc
Here is my users rights ->
![image](https://user-images.githubusercontent.com/19167503/58308586-42f68f80-7e02-11e9-9ab3-d0e6e2e95793.png)

And here is a file inside this folder that was created by a user. We see that there isn't the write mode on the file. So it's normal that collabora open in read only. But why the user which has write mode on this folder, when creating a new collabora doc, dont have the write permissions ??->
![image](https://user-images.githubusercontent.com/19167503/58308771-c3b58b80-7e02-11e9-8107-eeeec31861e4.png)

**Comment by langfingaz** [comment]
Ok, there we go. So the problem is not to open the files with Collabora, but to have the correct permissions inside Collabora. That explains why the fix of 4.0.1 didn't help you. Could you maybe edit your issue name according to that? (Sorry I'm not a developer, so I can't help further)

**Comment by thecube13** [comment]
Same problem. Is there any workaround?

**Comment by pierreozoux** [comment]
Looks related to https://github.com/nextcloud/groupfolders/issues/698

**Comment by juliushaertl** [comment]
Stumbled across this in the office online app today, and the fix from https://github.com/nextcloud/officeonline/pull/143.patch should apply similarly to Collabora so I'm moving this over to the richdocumets repo and will have a look into that.
